README.adoc

```diff
@@ -14,9 +14,9 @@ permalink: /guides/app-logging/
 :repo-description: Visit the https://kabanero.io/guides/{projectid}.html[website] for the rendered version of the guide.
 = Application Logging on OKD with Elasticsearch, FluentD, and Kibana
 
-In a Kubernetes environment, logs are constantly produced by the running pod processes. In order to effectively manage all the log data generated by a Kubernetes system, especially to prevent the loss of log data when a pod terminates, a log aggregation tool should be deployed on the user's OKD cluster to take control of all the logging activities. Modern log aggregating tools allow users to persist, search, and visualize their log data gathered from all the pods across their OKD clusters. There are a few log aggregating products available on the market that offer similar capabilities. This guide will walk through the process of deploying such a log aggregating tool called EFK stack using the ansible-playbook script provided by the OKD community.
+A Kubernetes environment constantly produce logs by its running pod processes. To effectively manage this log data and ensure no loss of log data occurs when a pod terminates, deploy a a log aggregation tool on the user's OKD cluster to control the logging activities. Log aggregation tools help users persist, search, and visualize the log data that is gathered from the pods across the OKD cluster. There are several log aggregation tools in the market that offer similar capabilities. This guide describes the process of deploying a log aggregating tool, **EFK (Elasticsearch, FluentD, and Kibana)**, using the ansible-playbook script provided by the OKD.
 
-OKD provides a preconfigured EFK(Elasticsearch, FluentD, and Kibana) stack, which a DevOps team uses to aggregate all container logs. You can install EFK stacks by using the ansible-playbook that resides in the https://github.com/openshift/openshift-ansible/tree/release-3.11[openshift-ansible] repository. After the installation completes, the EFK deployments reside inside the *openshift-logging* namespace of the OKD cluster.
+You use this preconfigured EFK stack to aggregate all container logs and use the the ansible-playbook that resides in the https://github.com/openshift/openshift-ansible/tree/release-3.11[openshift-ansible] repository to install EFK stacks on a OKD cluster. After a successful installation, the EFK deployments reside inside the *openshift-logging* namespace of the OKD cluster.
```

Comment:
19) First sentence - change phrase -- and use **the the** ansible-playbook -- TO -- and use the ansible-playbook --